### ISSUE TITLE
Cannot change shortcut <Super>q

### ISSUE THREAD

**Issue by kebeda** [micheleg/dash-to-dock#781]
The text entry box in the settings screen, behavior tab, "Use keyboard shortcuts to activate apps", "Show dock and application numbers" pop-up, "Shortcut for the options above" doesn't appear to be enabled. Although I can highlight the text entry box with my mouse, I cannot change the text `"<Super>q"` in any way. Tabbing through the dialog seems to skip this text entry box altogether (I can tab/modify the "Hide timeout(s)" box right below it however). Please fix this, as I'd like to use something other than `"<Super>q"` for this function. Alternatively, a gsettings command would be helpful.

**Comment by kebeda** [comment]
Here is a gsettings that will alter it (to Alt-d, in this case):

```
gsettings --schemadir ~/.local/share/gnome-shell/extensions/dash-to-dock@micxgx.gmail.com/schemas/ set org.gnome.shell.extensions.dash-to-dock shortcut "['<Alt>d']"
gsettings --schemadir ~/.local/share/gnome-shell/extensions/dash-to-dock@micxgx.gmail.com/schemas/ set org.gnome.shell.extensions.dash-to-dock shortcut-text '<Alt>d'

```
I would still like the settings dialog, fixed however.

**Comment by kebeda** [comment]
Thank you, Fran, I applied PR!782 to my machine and the text entry box now works as it should!

**Comment by micheleg** [comment]
Closed by #782 and and fix merged [082a76ca3c].
